Languages/Fortran/Makefile

```diff
@@ -62,7 +62,7 @@ all: $(EXE)
 %.o: %.f90
 	$(CC) $(FFLAGS) -c $< -o $@
 
-rocblas.mod: $(ROCBLAS_PATH)/include/rocblas_module.f90
+rocblas.mod: $(ROCBLAS_PATH)/include/rocblas/rocblas_module.f90
```

Comment:
It's unfortunate that ROCBLAS_PATH is typically just /opt/rocm, but using ROCM_PATH doesn't really make sense when using local builds. Just musing, isn't really an issue I guess.